sql/core/benchmarks/CSVBenchmarks-results.txt

```diff
@@ -0,0 +1,27 @@
+================================================================================================
+Benchmark to measure CSV read/write performance
+================================================================================================
+
+OpenJDK 64-Bit Server VM 1.8.0_163-b01 on Windows 7 6.1
+Intel64 Family 6 Model 94 Stepping 3, GenuineIntel
```

Comment:
Actually, `GHz` is missing here. So, it's hard to figure out what CPU is used here.
```
Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz [Family 6 Model 94 Stepping 3]
Intel(R) Core(TM) i7-6700T CPU @ 2.80GHz [Family 6 Model 94 Stepping 3]
Intel(R) Core(TM) i5-6600 CPU @ 3.30GHz [Family 6 Model 94 Stepping 3]
Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz [Family 6 Model 94 Stepping 3]
```

Reply:
This seems to be the limitation in Spark benchmark code itself.

Reply:
I made a PR to you. Could you review and merge https://github.com/heary-cao/spark/pull/2 ?